### PR TITLE
Recipes For Multi Lathe, Multi Autoclave, etc...

### PR DESCRIPTION
**Pull request by Volence**
Also all Item Pipe Casings (follow the same recipes for (Fluid) Pipe Casings, as well as the Pressure Containment Casings (they make up the Multi Autoclave).

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java [hunk 1]
```diff
@@ -1823,6 +1823,23 @@ public void run() {
...
+                        ItemList.AutoclaveLuV.get(2L),
+                        ItemList.Casing_Autoclave.get(4L),
+                        GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.LuV, 4L),
+                        GT_ModHandler.getModItem(GTPlusPlus.ID, "itemPlateLafiumCompound", 8L))
+                .fluidInputs(Materials.Polybenzimidazole.getMolten(8 * 144))
+                .itemOutputs(ItemList.Machine_Multi_Autoclave.get(1L)).duration(30 * SECONDS).eut(TierEU.RECIPE_LuV)
+                .addTo(assemblerRecipes);
+
+        // Industrial Autoclave
+        GT_Values.RA.stdBuilder()
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:1838]: this recipe looks like it produce nothing. Where is the output item/blocks ? @Volence

Comment (Volence) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:1838]: @Dream-Master Line 1840 has `.itemOutputs(ItemList.Casing_Autoclave.get(1L))`

Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:1838]: ah ok

**Review by Dream-Master — APPROVED** [review]

File: src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java [hunk 2]
```diff
@@ -692,6 +692,76 @@ public void run() {
...
+
+        // Fluxed Electrum
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 6),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.ElectrumFlux), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.ElectrumFlux), 'F',
+                        OrePrefixes.frameGt.get(Materials.ElectrumFlux) });
+
+        // Black Plutronium
```
Comment (EnderProyects) [inline src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java:757]: I'm on mobile so I can't do much, but is that name correct?

Comment (Volence) [reply src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java:757]: Fixed in this pr https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/950

**Pull request merged**